src/Microsoft.Framework.DesignTimeHost/ApplicationContext.cs

```diff
@@ -1113,7 +1115,7 @@ private DependencyInfo ResolveProjectDepencies(Project project, string configura
         {
             var cacheKey = Tuple.Create("DependencyInfo", project.Name, configuration, frameworkName);
 
-            return _cache.Get<DependencyInfo>(cacheKey, ctx =>
+            return CacheExtensions.Get(_cache, cacheKey, ctx =>
```

Comment:
It is, I think one of the VS automated refactoring operations might have changed it somehow. I'll revert that back. I certainly didn't actually type this in :P. Good catch :+1: